docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -302,75 +308,162 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
 
         :option-name:
 
-           * The name of the option/argument.
+            * The name of the option/argument.
 
         :description:
 
-           * Detailed explanation of what this option does. It should be written in full sentences.
+            * Detailed explanation of what this option does. It should be written in full sentences.
+
+        :version_added:
+        
+            * Only needed if this option was extended after the initial role release, in other words, this is greater than the top level 'version_added' field.
+            * This is a string, and not a float, for example, ``version_added: '2.1'``.
+            * In collections, this must be the collection version the option was added to. For example, ``version_added: 1.0.0``.
```

Comment:
To be complete `version_added` or `version_added_collection` should be allowed imo. This hasn't been supported, and invalid keys will give a warning soon (and be sanitized out of the spec), so this should be coordinated with https://github.com/ansible/ansible/pull/79657/files#r1063606171.

Reply:
`version_added` has been supported by antsibull-docs for role argument specs basically since the beginning.

`version_added_collection` hasn't been supported by antsibull-docs since it didn't make any sense so far; after all you cannot have docs fragments for role argument specs (so new options cannot come from other collections).

Reply:
(Actually for options antsibull-docs probably allows `version_added_collection` since it uses the same validation as for plugins.)

Reply:
So, I guess I can consider this resolved?

Reply:
I would say so.